portal/config/config.py

```diff
@@ -83,6 +83,7 @@ class BaseConfig(object):
     MAIL_USERNAME = os.environ.get('MAIL_USERNAME')
     MAIL_PASSWORD = os.environ.get('MAIL_PASSWORD')
     MAIL_USE_SSL = os.environ.get('MAIL_USE_SSL')
+    MAIL_USE_TLS = os.environ.get('MAIL_USE_TLS')
```

Comment:
These might need some boolean casting?
```suggestion
    MAIL_USE_SSL = os.environ.get('MAIL_USE_SSL', 'false').lower == "true"
    MAIL_USE_TLS = os.environ.get('MAIL_USE_TLS', 'false').lower == "true"
```